### DISCUSSION TITLE
npm broken after upgrading to node 15.0.1

### DISCUSSION THREAD

**Discussion by stevekennedyuk** [Homebrew/discussions#96] [Everyday usage]
npm upgrade -g
npm ERR! global requires an add or rm option

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/steve/.npm/_logs/2020-10-27T15_24_47_769Z-debug.log

Don't know if this is a Homebrew or npm directly issue

**Comment by SMillerDev** [comment]
This just seems like you're using it incorrectly. I'd ask on the NPM forum if something changed here for the latest version.

**Reply by stevekennedyuk** [reply]
hmm, nothing's changed my end, that's always how it used to work

**Reply by SMillerDev** [reply]
Yes, but new node version means an NPM update. And that could very well mean removed features
